core/scripts/install.sh

```diff
@@ -138,7 +138,6 @@ $settings = array(
     ),
     'install_configure_form' => array(
       'site_name' => $options['site-name'],
-      'site_mail' => $options['site-mail'],
```

Comment:
Looks like this will break the `--site-mail` option that is passed into `install.sh`. It should either still be accommodated, or removed from the list of options.